### PR TITLE
upgrading navigation var

### PR DESCRIPTION
**Pull request by Saarxp**

### CODE REVIEW

File: src/components/top-navbar/navbar.scss [hunk 1]
```diff
@@ -3,24 +3,34 @@
     display: flex;
     justify-content: space-around;
     align-items: center;
-    padding: 0.5rem 2rem;
+    padding: 1.4rem 1.4rem;
     background-color: #5d98f1;
```
Comment (vladbyk89) [inline src/components/top-navbar/navbar.scss:7]: The background color needs to be removed. See the new design.

File: src/components/top-navbar/LanguageDropdown.tsx [hunk 2]
```diff
@@ -55,7 +56,8 @@ export default function LanguageDropdown({ lang }: Readonly<{ lang: string }>) {
                         key={lang.lang.toString()}
                         onClick={() => handleSetLang(lang)}
                         style={{
-                            top: isOpen ? (i + 1) * 2 + "rem" : i + 1 + "rem",
+                            top: isOpen ? (i + 1) * 2 + "rem" : i + 1.3 + "rem",
+                            left: isOpen? (i + 1.3) + "rem" : i + 1.3 + "rem" ,
                             zIndex: isOpen ? 1 : -1,
                             opacity: isOpen ? 1 : 0,
                         }}
```
Comment (vladbyk89) [inline src/components/top-navbar/LanguageDropdown.tsx]: When opening the language dropdown menu, the flags need to align under each other.

File: src/components/icons/FreediIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+</svg>
+    );
+}
```
Comment (vladbyk89) [inline src/components/icons/FreediIcon.tsx:15]: Please use the same code format as in other files.

**Review by talyaron — APPROVED** [review]
Excellent work on your first PR! That was quick!

File: src/components/icons/FreediIcon.tsx (hunk 3 above)
Comment (Saarxp) [reply src/components/icons/FreediIcon.tsx:15]: done

File: src/components/top-navbar/LanguageDropdown.tsx (hunk 2 above)
Comment (Saarxp) [reply src/components/top-navbar/LanguageDropdown.tsx]: done

File: src/components/top-navbar/navbar.scss (hunk 1 above)
Comment (Saarxp) [reply src/components/top-navbar/navbar.scss:7]: done

**Pull request merged**